Changelog.md

```diff
@@ -4,6 +4,11 @@
 
 Nothing here yet.
 
+## v5.0.3 (2019-06-05)
```

Comment:
please, do not increase the version number yourself. It's the duty of the maintainer.
You can replace the "Nothing here yet." line by your changelog entry.

Reply:
Done